apps/cli/src/commands/create.ts

```diff
@@ -34,23 +35,23 @@ export default class CreateCommand extends Command {
                 "typescript",
             ],
         }),
-        branch: Flags.string({
-            description: `sunodo/sunodo-templates repository branch name to use`,
-            default: DEFAULT_TEMPLATES_BRANCH,
-        }),
+        repository: Flags.string({
+            description: `custom template repository to use`,
+            default: DEFAULT_TEMPLATE_REPOSITORY
+        })
     };
 
     private async download(
         template: string,
-        branch: string,
         out: string,
+        repository: string
     ): Promise<DownloadTemplateResult> {
         const sunodoProvider: TemplateProvider = async (input) => {
             return {
                 name: "sunodo",
                 subdir: input,
-                url: "https://github.com/sunodo/sunodo-templates",
-                tar: `https://codeload.github.com/sunodo/sunodo-templates/tar.gz/refs/heads/${branch}`,
+                url: repository,
+                tar: this.getTarUrl(repository),
```

Comment:
Keep the `sunodoProvider`  name here won't  make sense, maybe a more generic name like `repositoryProvider` or `githubProvider` since it may only work with github.